R/pkg/R/column.R

```diff
@@ -210,6 +210,18 @@ setMethod("cast",
             }
           })
 
+#' Specify multiple values
```

Comment:
Could you write a slightly more detailed description of what this operator does ? It will be great to also have a short example (similar to what we have for `cast`)